### PR TITLE
Added WebAuthn list

### PR DESCRIPTION
**Pull request by herrjemand**
**https://github.com/herrjemand/awesome-webauthn**

**It will help newcomers to start with integrating webauthn/fido2 into their existing authentication methods, and thus we will kill phishing and passwords.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

I solemnly swear that I fully read it.
 
## Requirements for your pull request

- **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).** Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration.
- I have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with the below requirements.

## Requirements for your Awesome list

- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- It's the result of hard work and the best I could possibly produce.
- Non-generated Markdown file in a GitHub repo.
- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- The image should be high-DPI. Set it to maximum half the width of the original image.
- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
	- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
	- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
	- If you use a license badge, it should be SVG, not PNG.
- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.

Go to the top and read it again.

### CODE REVIEW

**Comment by samber** [comment]
In order to match guideline, you should fix those errors:

```
✖ Linting

  README.md:18:3
  ✖   1:1    Missing file contributing.md                            awesome/contributing
  ✖   1:1    Missing or invalid Table of Contents                    awesome/toc
  ✖   1:21   Invalid badge source                                    awesome/badge
  ✖   5:3    Invalid list item link URL                              awesome/list-item
  ✖   6:5    Invalid list item link URL                              awesome/list-item
  ✖   8:3    Invalid list item link URL                              awesome/list-item
  ✖   9:5    Invalid list item link URL                              awesome/list-item
  ✖  10:5    Invalid list item link URL                              awesome/list-item
  ✖  11:5    Invalid list item link URL                              awesome/list-item
  ✖  12:5    Invalid list item link URL                              awesome/list-item
  ✖  14:3    Invalid list item link URL                              awesome/list-item
  ✖  15:3    Invalid list item link URL                              awesome/list-item
  ✖  18:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  18:72   List item description must end with proper punctuation  awesome/list-item
  ✖  19:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  19:110  List item description must end with proper punctuation  awesome/list-item
  ✖  20:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  20:77   List item description must end with proper punctuation  awesome/list-item
  ✖  21:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  21:129  List item description must end with proper punctuation  awesome/list-item
  ✖  22:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  22:160  List item description must end with proper punctuation  awesome/list-item
  ✖  23:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  23:47   List item description must end with proper punctuation  awesome/list-item
  ✖  24:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  24:145  List item description must end with proper punctuation  awesome/list-item
  ✖  25:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  26:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  26:134  List item description must end with proper punctuation  awesome/list-item
  ✖  30:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  31:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  32:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  33:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  33:83   List item description must end with proper punctuation  awesome/list-item
  ✖  34:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  34:69   List item description must end with proper punctuation  awesome/list-item
  ✖  35:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  36:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  36:66   List item description must end with proper punctuation  awesome/list-item
  ✖  37:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  37:74   List item description must end with proper punctuation  awesome/list-item
  ✖  40:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  40:66   List item description must end with proper punctuation  awesome/list-item
  ✖  41:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  42:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  45:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  46:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  48:1    Don’t use multiple top level headings (1:1)             no-multiple-toplevel-headings
  ✖  50:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  51:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  52:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  55:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  56:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  57:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  57:47   List item description must end with proper punctuation  awesome/list-item
  ✖  58:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  59:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  59:81   List item description must end with proper punctuation  awesome/list-item
  ✖  60:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  60:50   List item description must end with proper punctuation  awesome/list-item
  ✖  61:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  62:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  65:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  66:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  69:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  69:81   List item description must end with proper punctuation  awesome/list-item
  ✖  71:2    Remove empty section: "Other"                           no-empty-sections
  ✖  72:3    Incorrect indentation before bullet: remove 1 space     list-item-bullet-indent
  ✖  72:79   List item description must end with proper punctuation  awesome/list-item

  69 errors
```

- Rename `CONTRIBUTIONS.md` to `contributing.md`.
- Add a "Contents" section title
- no more than one h1
- h2 and h3 must be represented in table of content in same order
- In `demos`, `server libs`, `client libs`, `hardware`, `tutorials`, `articles`, `slides`, `books` and `other` sections, you should remove first space

To improve quality of awesome lists, Sindresorhus made a simple linter.
You can embed it into a Travis CI pipeline easily: https://github.com/sindresorhus/awesome-lint

###### package.json

```json
{
	"scripts": {
		"test": "awesome-lint"
	},
	"devDependencies": {
		"awesome-lint": "*"
	}
}
```

###### .travis.yml

```
language: node_js
node_js:
  - 'node'
```

File: readme.md [hunk 1]
```diff
@@ -448,7 +448,7 @@
 - [Vehicle Security and Car Hacking](https://github.com/jaredthecoder/awesome-vehicle-security#readme)
 - [Web Security](https://github.com/qazbnm456/awesome-web-security#readme) - Security of web apps & services.
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
-
+- [WebAuthn](https://github.com/herrjemand/awesome-webauthn#readme) - A new W3C standard for phishing proof passwordless authentication
```
Comment (sindresorhus) [inline readme.md]:
```suggestion
- [WebAuthn](https://github.com/herrjemand/awesome-webauthn#readme) - A new W3C standard for phishing-proof passwordless authentication.
```

**Comment by sindresorhus** [comment]
> * This pull request has a descriptive title in the format `Add Name of List` (Example: `Add Swift`), not `Update readme.md` or `Add awesome list`.

**Comment by sindresorhus** [comment]
@herrjemand Ping

**Comment by sindresorhus** [comment]
> * **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Comment by herrjemand** [comment]
@sindresorhus Sorry, I have not seen your message. Could you reopen and I will fix all the issues

**Pull request closed without merging**